src/app/shared/footer/footer.ts

```diff
@@ -14,6 +16,7 @@ export class Footer {
 
 
 @NgModule({
+  imports: [RouterModule.forRoot(MATERIAL_DOCS_ROUTES)],
```

Comment:
Please use `RouterModule#forChild` instead of `RouterModule#forRoot` on submodules/lazy-loaded modules (as indicated in the [API docs for `RouterModule`](https://angular.io/api/router/RouterModule#usage-notes)).

```suggestion
  imports: [RouterModule.forChild(MATERIAL_DOCS_ROUTES)],
```